tests/Jenkins/dirac-cfg-update-services.py

```diff
@@ -48,38 +51,43 @@
 
 csAPI = CSAPI()
 
+fc = "FileCatalog"
+multiFC = "MultiVOFileCatalog"
+
+if os.environ["TEST_HTTPS"] == "Yes":
```

Comment:
```suggestion
if os.environ.get("TEST_HTTPS", "Yes") == "Yes":
```
In the bash code you account for the possibility that TEST_HTTPS is not set, I think.